frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx

```diff
@@ -74,27 +51,21 @@ export const PoolResourceComponent: React.FC<ProvisionerProps> = ({ onParamChang
   const poolDropdownItems = _.reduce(
     poolData,
     (res, pool: StoragePoolKind) => {
-      if (
-        usedPool.indexOf(pool?.metadata.name) === -1 &&
-        pool?.status?.phase === POOL_STATE.READY &&
```

Comment:
I'll say we can still have this condition till the time the Pool is not in the Ready phase.

Reply:
ack

Reply:
May be we would consider to disable the dropdown by setting `isDisabled` on the DropdownItem @cloudbehl @a2batic .
This will list the pools but wont allow selection thereby fulfilling the BZ expectation.